File: Arnav Sharma/Assignment-1/negative_array.java
package com.array;
import java.util.Scanner;

public class negative_array {
    public static void main(String[] args) {
        int n;
        Scanner sc = new Scanner(System.in);
        System.out.println("enter the no of elements");
        n = sc.nextInt();

        int[] arr = new  int[15];
        System.out.println("Enter the elements");
        for (int i=0; i<n; i++){
            arr[i] = sc.nextInt();
        }
        System.out.println("Negative elements are: ");
        {
            for(int i=0;i<n;i++){
                if(arr[i]<0)
                    System.out.println(arr[i]);
            }
        }
    }
}

File: Aishwarya Kadu/assignment 1/negative array.c
// C program to print all negative elements in array
#include <stdio.h>
#define MAX_SIZE 100 e
int main()
{
    int arr[MAX_SIZE];
    int i, N;
    printf("Enter size of the array : ");
    scanf("%d", &N);
    printf("Enter elements in array : ");
    for(i=0; i<N; i++)
    {
        scanf("%d", &arr[i]);
    }

    printf("\nAll negative elements in array are : ");
    for(i=0; i<N; i++)
    {
        if(arr[i] < 0)
        {
            printf("%d\t", arr[i]);
        }
    }
    return 0;
}